feature/tabsflow/home/src/main/java/com/epmedu/animeal/home/presentation/ui/map/RouteView.kt

```diff
@@ -115,4 +124,25 @@ private fun fetchRoute(
             onRouteResult = onRouteResult
         )
     }
+}
+
+private fun getMapBoxNavigationInitOptions(
+    mapView: MapView,
+    state: HomeState
+): MapBoxRouteInitOptions {
+    val routeLineResources = RouteLineResources.Builder()
+        .originWaypointIcon(R.drawable.ic_your_location)
```

Comment:
This icon should be set for user location when there is no route, please see [the design](https://www.figma.com/file/ihPSDMVPPlB6lIstYERxg5/Doggo?type=design&node-id=1776-1545&mode=dev)

Reply:
Got it! Thank you for the observation! I got it refactored :)